packages/sdks/overrides/react-native/src/index.ts

```diff
@@ -12,6 +12,7 @@ export { default as Section } from './blocks/section.lite';
 export { default as Fragment } from './blocks/fragment.lite';
 export { default as RenderContent } from './components/render-content.lite';
 
+export * from './functions/is-editing';
```

Comment:
Did I need to add this line to all of the `overrides/*` files as well? I did it to follow the convention but I don't fully understand what's happening here.

Reply:
yeah probably, overrides are a heavy handed way to totally replace a file per output target